setup.py

```diff
@@ -93,7 +93,6 @@
     'types-MarkupSafe~=1.1',
     'types-pkg-resources~=0.1.3',
     'types-typed-ast~=1.4.2',
-    'types-requests~=2.25.6',
```

Comment:
`requests-xml` has `requests` as a transitive dependency, so we have to keep this, I think.

Reply:
Right, actually I added it when I was using `requests` in the test - but keeping it is also fine, in case someone needs requests in the future.